netsim/modules/vlan.py

```diff
@@ -650,7 +656,7 @@ def create_vlan_links(link: Box, v_attr: Box, topology: Box) -> None:
         link_data.vlan.mode = 'route'
         for intf in link_data.interfaces:
           intf.vlan.mode = 'route'
-      else:
+      elif prefix is not None:
```

Comment:
Not sure we ever got here in the tests, but this effectively blocks using empty "prefix:" on a VLAN. Changing to a slightly safer approach.